utils/polus-python-template/cookiecutter.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
More comments needed to describe what the schema does, and which parts do what.

Reply:
You can't really do comments in JSON. At best what you can do is add extra fields to the json, but that might mess up WIPP.

https://stackoverflow.com/questions/244777/can-comments-be-used-in-json